src/server/utils/pdf/parser.ts

```diff
@@ -5,42 +5,44 @@ import { getInnovationInfoEndpointDTO, sectionType } from '@modules/stores/innov
 import { AllSectionsOutboundPayloadType, getAllSectionsSummary } from '@modules/stores/innovation/innovation.config';
 
 import { ENVIRONMENT } from '../../config/constants.config';
-import { PDFGenerator } from './PDFGenerator';
-import { PDFGeneratorInnovationNotFoundError, PDFGeneratorParserError, PDFGeneratorSectionsNotFoundError } from '../errors';
 
+import { PDFGeneratorParserError, PDFGeneratorSectionsNotFoundError } from '../errors';
 
-export const getSections = async (innovationId: string, userId: string, config: any): Promise<{ section: sectionType, data: MappedObjectType }[]> => {
-  const url = `${ENVIRONMENT.API_URL}/api/innovators/${userId}/innovations/${innovationId}/sections`;
+
+export const getSections = async (innovationId: string, config: any): Promise<{ section: sectionType, data: MappedObjectType }[]> => {
+  //const url = `${ENVIRONMENT.API_URL}/api/innovations/v1/${innovationId}/all-sections`;
+  const url = `${ENVIRONMENT.LOCAL_API_INNOVATIONS_BASE_URL}/api/v1/${innovationId}/all-sections`;
```

Comment:
This should be comment, right? Same on another places.

Reply:
yep!